### PR TITLE
Display user feed

### PR DESCRIPTION
**Pull request by ZohaibManzoor00**
Lightmode: 
Screenshot (255)

Darkmode: 
Screenshot (254)

### CODE REVIEW

**Review by ntan03 — DISMISSED** [review]

File: src/shared/components/footer/Footer.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ function Footer() {
   } = useTheme();
 
   return (
-    <Box sx={{ position: "absolute", bottom: "0", width: "100%" }}>
+    // <Box sx={{ position: "absolute", bottom: "0", width: "100%" }}>
```
Comment (DLiSWE) [inline src/shared/components/footer/Footer.tsx:12]: Comment?

File: src/shared/components/footer/Footer.tsx [hunk 2]
```diff
@@ -84,7 +84,7 @@ function Footer() {
           </Typography>
         </Box>
       </Box>
-    </Box>
+    // </Box>
```
Comment (DLiSWE) [inline src/shared/components/footer/Footer.tsx:87]: Here too

File: src/shared/components/post/PostFooter.tsx [hunk 3]
```diff
@@ -68,18 +69,24 @@ function PostFooter({ id, userId }: BasePostFooterType) {
...
+          <Typography sx={{ color }}>
+            {displayLikeCount} {displayLikeCount === 1 ? "Like" : "Likes"}
+          </Typography>
         </Box>
         <Box component="span" sx={{ display: "flex", alignItems: "center" }}>
-          {commentCount === 1 ? (
+          {/* {comments.length === 1 ? (
             <Typography sx={{color}}>{commentCount} Comment</Typography>
           ) : (
             <Typography sx={{color}}>{commentCount} Comments</Typography>
```
Comment (DLiSWE) [inline src/shared/components/post/PostFooter.tsx:85]: Are you saving this for later?

File: src/shared/components/footer/Footer.tsx (hunk 1 above)
Comment (ZohaibManzoor00) [reply src/shared/components/footer/Footer.tsx]: Good catch

File: src/shared/components/post/PostFooter.tsx (hunk 3 above)
Comment (ZohaibManzoor00) [reply src/shared/components/post/PostFooter.tsx]: Me and Nelson were using live share and forgot to delete this. Good catch!

**Review by ntan03 — APPROVED** [review]

**Pull request merged**